tasks/src/robocorp/tasks/cli.py

```diff
@@ -9,35 +9,54 @@
 code which must be automatically logged is not imported prior the the `cli.main`
 call.
 """
+
 import os
 import sys
 import warnings
+from importlib.metadata import PackageNotFoundError, version as get_version
 from typing import List, Optional, Protocol
 
+from packaging import version as version_parser
+
 
-def inject_truststore():
-    # Use certificates from native storage (if `truststore` installed)
+def _inject_truststore():
```

Comment:
Making it private as there's no reason/effect at all to have it publicly available.